CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * [BUGFIX] No-longer-needed ingester operations for queries triggered by queriers and rulers are now canceled. #3178
 * [BUGFIX] Ruler: directories in the configured `rules-path` will be removed on startup and shutdown in order to ensure they don't persist between runs. #3195
 * [BUGFIX] Handle hash-collisions in the query path. #3192
+* [BUGFIX] Experimental Alertmanager: Do not allow empty Alertmanager configurations or bad template filenames to be submitted through the configuration API. #3185
```

Comment:
I would make it clear it's the API, because the alertmanager itself is not experimental.

```suggestion
* [BUGFIX] Experimental Alertmanager API: Do not allow empty Alertmanager configurations or bad template filenames to be submitted through the configuration API. #3185
```